scripts/docker/entrypoint_prod.sh

```diff
@@ -283,8 +283,8 @@ if [[ "${CONNECTION_CHECK_MAX_COUNT}" -gt "0" ]]; then
     wait_for_airflow_db
 fi
 
-if [[ -n "${_AIRFLOW_DB_UPGRADE=}" ]] ; then
-    upgrade_db
+if [[ -n "${_AIRFLOW_DB_UPGRADE=}" ]] || [[ -n "${_AIRFLOW_DB_MIGRATE=}" ]] ; then
```

Comment:
Made this change so that its backward compatible

Reply:
Should we add a warning if the `UPGRADE` environment variable is used?

Reply:
Yes. Added a warning.